beeline-core/src/main/java/io/honeycomb/beeline/builder/README.md

```diff
@@ -0,0 +1,66 @@
+# Building Beelines with the BeelineBuilder
+BeelineBuilder is an attempt to create a unified API for Beelines to simplify user configuration between the Beeline and the HoneyClient.
```

Comment:
Let's take out "is an attempt to create" and just say "is a unified API ..."

Reply:
Updated.